src/noit_module.c

```diff
@@ -268,6 +277,15 @@ void noit_module_init() {
       continue;
     }
 
+    /* Check if we intend to allow this module. */
+    if(module_match) {
+      int ovector[30];
+      if(pcre_exec(module_match, NULL, module_name, strlen(module_name), 0, 0, ovector, 30) < 0) {
+        mtevL(mtev_notice, "skipping check module '%s' due to NOIT_MODULES.\n", module_name);
```

Comment:
I might say "due to NOIT_MODULES environment variable." just to be crystal clear what is causing the skip.